eox_hooks/actions.py

```diff
@@ -0,0 +1,70 @@
+import requests
+from django.conf import settings
+
+from eox_hooks.serializers import CertificateSerializer,UserSerializer
+from eox_hooks.utils import _get_action_settings, _get_course, flatten
+from lms.djangoapps.course_api.serializers import CourseDetailSerializer
```

Comment:
I think that we need to create a edxapp_wrapper module in order to use the models and serializers from edx-platform

Reply:
Ok, done